index.html

```diff
@@ -47,6 +47,157 @@ <h3>Let's CONNECT</h3>
+                <div class="work-technology">
+                    <ul>
+                        <li class="">CANOPY</li>
+                        <li class=""><img src="assets/counter.png" alt=""></li>
+                        <li class="tech">Back End Dev</li>
+                        <li class=""><img src="assets/Counter.png" alt=""></li>
+                        <li class="date">2015</li>
+                    </ul>
+                </div>
+                <div class="work-details">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-skills">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
+            </div>
+        </section>
+
+        <section class="about">
+            <div class="about-container">
+                <div class="personal-info-title">
+              <h1><a href="#" class="header-info about-header">About MySelf</a></h1>
+              </div>
+              <div class="about-info">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+              </div>
+              <div class="personal-social-info">
+                <h3>Let's CONNECT</h3>
+                <div class="social-media">
+                    <ul>
+                    <li><a href="#"><img src="assets/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="#"><img src="assets/in.png" alt="linkdin"></a></li>
+                    <li><a href="#"><img src="assets/m.svg" alt=""></a></li>
+                    <li><a href="#"><img src="assets/git.svg" alt="github"></a></li>
+                    <li><a href="#"><img src="assets/peace.svg" alt="peace"></a></li>
+                </ul>
+                </div>
+                  <div class="work-button">
+                      <button>Get my resume</button>
+                      </div>
+                    <div class="languages">
+                        <h2>Languages</h2>
+                        <ul>
+                            <li><img src="assets/js.png" alt=""><p>Javascript</p></li>
+                            <li><img src="assets/html.png" alt=""><p>HTML</p></li>
+                            <li><img src="assets/css.png" alt=""><p>CSS</p></li>
+                        </ul>
+                    </div>
+                    <div class="frameworker">
+                        <h2>FrameWork</h2>
+                    </div>
+                    <div class="skills">
+                        <h2>Skills</h2>
+                    </div>
+            </div>
+            </div>
```

Comment:
- [ ] Kindly add the right icons to the languages, skills and frameworks div 👍🏿 

 Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_12](https://user-images.githubusercontent.com/69638013/159021445-ec69ddb9-54af-4fbc-9442-12ea040a9b3a.png)  | ![iconn](https://user-images.githubusercontent.com/69638013/159021138-4e044839-d926-4142-9b21-86604e597a12.png)


- [ ] It would be nice to add the border-radius to li's in the languages section
- [ ] It would be nice to add the same font family, colors and font size to your design, this will make your project look more similar to the Figma design